src/main/kotlin/dev/haarolean/gnutardbot/abilities/CheckLinksAbility.kt

```diff
@@ -1,68 +1,63 @@
 package dev.haarolean.gnutardbot.abilities
 
 import dev.haarolean.gnutardbot.TardBot
-import dev.haarolean.gnutardbot.util.MemberUtils
-import org.apache.commons.lang3.StringUtils
-import org.springframework.beans.factory.config.BeanDefinition
-import org.springframework.context.annotation.Scope
-import org.springframework.stereotype.Component
-import org.telegram.abilitybots.api.objects.*
+import org.telegram.abilitybots.api.objects.MessageContext
 import org.telegram.abilitybots.api.util.AbilityUtils
 import org.telegram.telegrambots.meta.api.methods.groupadministration.BanChatMember
 import org.telegram.telegrambots.meta.api.methods.updatingmessages.DeleteMessage
 import org.telegram.telegrambots.meta.api.objects.Message
+import dev.haarolean.gnutardbot.util.MemberUtils
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage
+import java.util.regex.Pattern
 
-@Component
-@Scope(BeanDefinition.SCOPE_PROTOTYPE)
```

Comment:
different abilities now have a different set of annotations or none at all. Let's make them identical, please